README.md

```diff
@@ -2128,10 +2128,10 @@ Gets the value at path of object.
   console.log(result);
   // output: 3
 
-  // Native (ES6 - IE not supported)
```

Comment:
Could you keep both instead of deleting the old one?

Reply:
Yeah of course. I just noticed that the current ES6 example doesn't work correctly. Don't we need to add a default value for each nested object when destructuring it? 🤔
![image](https://user-images.githubusercontent.com/7090566/80130061-4eb17f80-8598-11ea-8ca0-68a18093ff7e.png)